ene-jun-2022/SalvadorPaniagua/Practica_1.java

```diff
@@ -0,0 +1,22 @@
+package algoritmos.pkg2022;
+import javax.swing.JOptionPane;
+
+
+public class Practica_1 {
+
+
+public static void main(String[] args) {
+    int x;
+    int y;
+    int suma;
+   
+                JOptionPane.showMessageDialog(null, "Primer numero");
+                x =Integer.parseInt(JOptionPane.showInputDialog(null,"Numero 1"));
```

Comment:
En lugar de utilizar `JOptionPane` para mostrar una interfaz gráfica, utiliza `Scanner` con `std.in` para poder leer de la terminal directamente.